kwsStyle.cxx

```diff
@@ -284,67 +284,67 @@ int main(int argc, const char **argv)
       return 0;
       }
     file.seekg(0,std::ios::end);
-    unsigned long fileSize = static_cast<unsigned long> (file.tellg());
+    std::streamsize fileSize = file.tellg();
     file.seekg(0,std::ios::beg);
 
     char* buf = new char[fileSize+1];
-    file.read(buf, static_cast<std::streamsize>(fileSize));
+    file.read(buf, fileSize);
     buf[fileSize] = 0;
     std::string buffer(buf);
-    buffer.resize(fileSize);
+    buffer.resize(static_cast<size_t>(fileSize));
     delete [] buf;
 
-    long int start = 0;
-    auto pos = static_cast<long int>(buffer.find("\n", static_cast<size_t>(start)));
+    size_t start = 0;
+    auto pos = buffer.find("\n", start);
     do
       {
       std::string line = "";
 
       if(pos == -1)
         {
-        line = buffer.substr(static_cast<size_t>(start),buffer.length()- static_cast<size_t>(start));
-        pos = static_cast<long>(fileSize); // we stop
+        line = buffer.substr(start,buffer.length()- start);
+        pos = static_cast<size_t>(fileSize); // we stop
```

Comment:
No cast should be needed here now, correct?

Reply:
fileSize is std::streamsize. I could change it to size_t but i had to cast it some where else